### PR TITLE
Improving Spanish intents, adding synonyms

### PR DESCRIPTION
**Pull request by davefx**

### CODE REVIEW

File: sentences/es/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -4,7 +4,8 @@ intents:
     data:
       - sentences:
         - "cuál es [la] <temp>"
+        - "(cuánta|cuánto) <temp> hace"
         - "qué <temp> hace"
-        - "cuál es [la] <temp> <area>"
+        - "cuál es [la|el] <temp> en <area>"
```
Comment (balloob) [inline sentences/es/climate_HassClimateGetTemperature.yaml:9]: We can remove `[la|el]` as it's already included in `<temp>`

Comment (balloob) [reply sentences/es/climate_HassClimateGetTemperature.yaml:9]:
```suggestion
        - "cuál es <temp> en <area>"
```

File: sentences/es/climate_HassClimateGetTemperature.yaml [hunk 2]
```diff
@@ -4,7 +4,8 @@ intents:
     data:
       - sentences:
         - "cuál es [la] <temp>"
```
Comment (balloob) [inline sentences/es/climate_HassClimateGetTemperature.yaml:6]:
```suggestion
        - "cuál es <temp>"
```

**Comment by balloob** [comment]
- Please keep contributions to a single domain per PR. So only `light`, `homeassistant` or `cover`
- Instead of copy/pasting the same word combinations to each file, you can add an expansion rule to `_common.yaml`

File: sentences/es/climate_HassClimateSetTemperature.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ intents:
   HassClimateSetTemperature:
     data:
       - sentences:
-        - "pon <temp> a <temperature>"
-        - "pon <temp> <area> a <temperature>"
-        - "pon <area> <temp> a <temperature>"
+        - "(pon|poner|establece|establecer|ajusta|ajustar) <temp> a <temperature>"
```
Comment (balloob) [inline sentences/es/climate_HassClimateSetTemperature.yaml:6]: Consider moving this to an expansion rule

File: sentences/es/cover_HassCloseCover.yaml [hunk 4]
```diff
@@ -3,14 +3,14 @@ intents:
   HassCloseCover:
     data:
       - sentences:
-          - "cierra <name>"
-          - "cierra <name> <area>"
+          - "(cierra|cerrar|baja|bajar) <name>"
```
Comment (jfisbein) [inline sentences/es/cover_HassCloseCover.yaml:6]: Consider moving this to an expansion rule

File: sentences/es/cover_HassOpenCover.yaml [hunk 5]
```diff
@@ -3,14 +3,14 @@ intents:
   HassOpenCover:
     data:
       - sentences:
-          - "abre <name>"
-          - "abre <name> <area>"
+          - "(abre|abrir|sube|subir) <name>"
```
Comment (jfisbein) [inline sentences/es/cover_HassOpenCover.yaml:6]: Consider moving this to an expansion rule

File: sentences/es/fan_HassTurnOff.yaml [hunk 6]
```diff
@@ -3,7 +3,7 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - "apaga [todos] [los] ventiladores <area>"
+          - "<apaga> [todos] [los] ventiladores [(de|en)] <area>"
```
Comment (jfisbein) [inline sentences/es/fan_HassTurnOff.yaml:6]: the `[(de|en)]` is not needed as it's already included in the `<area>` expansion rule

File: sentences/es/fan_HassTurnOn.yaml [hunk 7]
```diff
@@ -3,7 +3,7 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - "enciende [todos] [los] ventiladores <area>"
+          - "<enciende> [todos] [los] ventiladores [(de|en)] <area>"
```
Comment (jfisbein) [inline sentences/es/fan_HassTurnOn.yaml:6]: the [(de|en)] is not needed as it's already included in the <area> expansion rule
```suggestion
          - "<enciende> [todos] [los] ventiladores <area>"
```

File: sentences/es/light_HassLightSet.yaml [hunk 8]
```diff
@@ -4,17 +4,17 @@ intents:
     data:
       # brightness
       - sentences:
-          - "pon [el] brillo de <name> al <brightness>"
-          - "pon <name> (al | con) brillo <brightness>"
-          - "pon <name> a <brightness>"
+          - "(pon|poner|sube|subir|baja|bajar|establece|establecer) [el] brillo de <name> al <brightness>"
```
Comment (jfisbein) [inline sentences/es/light_HassLightSet.yaml:7]: Consider moving this to an expansion rule

File: sentences/es/climate_HassClimateSetTemperature.yaml (hunk 3 above)
Comment (davefx) [reply sentences/es/climate_HassClimateSetTemperature.yaml]: Done

File: sentences/es/climate_HassClimateGetTemperature.yaml [hunk 9]
```diff
@@ -3,8 +3,9 @@ intents:
   HassClimateGetTemperature:
     data:
       - sentences:
-        - "cuál es [la] <temp>"
+        - "cuál es <temp>"
+        - "(cuánta|cuánto) <temp> hace"
         - "qué <temp> hace"
-        - "cuál es [la] <temp> <area>"
+        - "cuál es <temp> en <area>"
```
[the remark this answers is not in the document]
Comment (davefx) [reply sentences/es/climate_HassClimateGetTemperature.yaml]: Fixed

File: sentences/es/cover_HassCloseCover.yaml (hunk 4 above)
Comment (davefx) [reply sentences/es/cover_HassCloseCover.yaml]: Done

File: sentences/es/cover_HassOpenCover.yaml (hunk 5 above)
Comment (davefx) [reply sentences/es/cover_HassOpenCover.yaml]: Done

File: sentences/es/fan_HassTurnOff.yaml (hunk 6 above)
Comment (davefx) [reply sentences/es/fan_HassTurnOff.yaml]: Fixed

File: sentences/es/fan_HassTurnOn.yaml (hunk 7 above)
Comment (davefx) [reply sentences/es/fan_HassTurnOn.yaml]: Fixed

File: sentences/es/light_HassLightSet.yaml (hunk 8 above)
Comment (davefx) [reply sentences/es/light_HassLightSet.yaml]: Done

**Review by cibernox — APPROVED** [review]

**Pull request merged**